src/Symfony/Component/Ldap/Adapter/EntryManagerInterface.php

```diff
@@ -40,4 +40,13 @@ public function update(Entry $entry);
      * @param Entry $entry
      */
     public function remove(Entry $entry);
+
+    /**
+     * Renames an entry on the Ldap server.
+     *
+     * @param Entry  $entry
+     * @param string $newRdn
+     * @param bool   $removeOldRdn
+     */
+    public function rename(Entry $entry, $newRdn, $removeOldRdn = true);
```

Comment:
`move()` is something else, and not related to `rename()`.

So the correct interface would be something like `RenameEntryInterface`. We could add `MoveEntryInterface` at a later time, when we actually implement moving an entry to a new parent tree.